src/howard/howard/templates/howard/certificate.html

```diff
@@ -17,7 +18,11 @@
             <img src="{{ debug | yesno:",file://" }}{% static "howard/logo-fun.png" %}" />
           </div>
           <div class="organization">
-            <img src="{{ debug | yesno:",file://" }}{{ course.organization.logo }}" />
+              {% if course.organization.logo|is_path %}
```

Comment:
Simple suggestion which I find more readable:

```suggestion
              {% if course.organization.logo | is_path %}
```